staging/src/k8s.io/apiserver/pkg/registry/rest/create.go

```diff
@@ -151,3 +152,24 @@ type NamespaceScopedStrategy interface {
 	// NamespaceScoped returns if the object must be in a namespace.
 	NamespaceScoped() bool
 }
+
+// AdmissionToValidateObjectFunc converts validating admission to a rest validate object func
+func AdmissionToValidateObjectFunc(validatingAdmission admission.ValidationInterface, staticAttributes admission.Attributes) ValidateObjectFunc {
+	return func(obj runtime.Object) error {
+		finalAttributes := admission.NewAttributesRecord(
+			obj,
+			staticAttributes.GetOldObject(),
+			staticAttributes.GetKind(),
+			staticAttributes.GetNamespace(),
+			staticAttributes.GetName(),
+			staticAttributes.GetResource(),
+			staticAttributes.GetSubresource(),
+			staticAttributes.GetOperation(),
+			staticAttributes.GetUserInfo(),
+		)
+		if !validatingAdmission.Handles(finalAttributes.GetOperation()) {
```

Comment:
this check could be much earlier in the call flow, couldn't it?

Reply:
at the very least outside of the closure.

Reply:
> at the very least outside of the closure.

It could be.  I don't know that it would make it easier to move it outside the closure since the result affects the returned function.  Just preference?

Reply:
Just preference and a little little performance optimization. Anyway, not more than a nit.

Reply:
> Just preference and a little little performance optimization. Anyway, not more than a nit.

Ok.  I've got no issue with moving it.  Just making sure.